src/containers/CreateAccount/CreateAccount.jsx

```diff
@@ -1,11 +1,129 @@
-import React from 'react'
+import React, { useState } from 'react';
+import "./CreateAccount.scss";
+import InputField from "../../components/InputField/InputField";
+import Button from "../../components/Button/Button";
+import BackButton from "../../components/BackButton/BackButton";
+import SocialLogin from "../../components/SocialLogin/SocialLogin";
+import { createUserWithEmailAndPassword } from "firebase/auth";
+import { useNavigate } from 'react-router-dom';
+import facebookIcon from "../../assets/images/facebook-logo.svg";
+import googleIcon from "../../assets/images/google-logo.svg";
+import appleIcon from "../../assets/images/apple-logo.svg";
 
-// path: /create-account
 
-const CreateAccount = () => {
+const CreateAccount = ({auth, user, handleInputChange, setUser}) => {
+  const [page, setPage] = useState(1);
+  const [passwordConfirm, setPasswordConfirm] = useState("");
+  const navigate = useNavigate();
+
+  const switchPage = () => {
+    if (user.firstName === "" || user.lastName === "") {
+      return
+    }
+  
+    if (page === 1) {
+      setPage(2);
+    } else {
+      setPage(1);
+    }
+  };
+
+  const handlePasswordConfirmChange = (event) => {
+    setPasswordConfirm(event.target.value);
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    createUserWithEmailAndPassword(auth, user.email, user.password)
+      .then((userCredential) => {
+        console.log('Success')
+        const user = userCredential.user;
```

Comment:
You should be able to set the first name & last name of the user after it has been created. At the moment it will eventually get lost.

```js
import { auth } from "../../firebase";
import { updateProfile } from "firebase/auth";
```
```js
      updateProfile(auth.currentUser, {
        firstName: user.firstName ,
        lastName : user.lastName,
      })
```